.dockstore.yml

```diff
@@ -117,6 +117,7 @@ workflows:
        branches:
          - master
          - ah_var_store
+         - rc-vs-317-remove-excess-headers
```

Comment:
Do we get rid of these before we merge the PR?

Reply:
Ideally yes, but with tests taking so long, sometimes we dont. It would be great to do a full clean up of all the extra branches in dockstore too